Fabric.Authorization.AccessControl/src/app/services/interceptors/fabric-http-error-handler-interceptor.service.ts

```diff
@@ -41,7 +41,7 @@ export class FabricHttpErrorHandlerInterceptorService {
             )}`
           );
 
-          this.showAlert(response, response.statusText);
+          this.showAlert(response, response.error.message || response.statusText);
```

Comment:
@johnrulon I don't think so in this case, but it may not hurt to add that check. This scenario handles errors from our backend services which should have that object populated for all cases. I'll look to see if we should add it.

Reply:
This block of code is below the logic I added, so I think we're ok:

```
return observableThrowError(
          new Exception(response.status, response.error.message || JSON.stringify(response.error))
        );
```